### PR TITLE
Add intro blog post

### PR DESCRIPTION
**Pull request by ojacques**
Add intro blog post, explaining the context of the Online DevOps Dojo.

### CODE REVIEW

**Comment by ojacques** [comment]
Thanks @pberthonneau ! Took all the changes into account. Agreed there is duplication with DevOps@DXC, and intent is a self contained blog post.

File: blog/_posts/2020-04-14-introducing-online-devops-dojo.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+The modules tell the story of a fictious company with its employees as they go through their DevOps journey.
+
+![](/images/onceuponatime.jpg)
+
+Let us introduce you to some of the team members:
+
+![Brenda](/images/brenda.png) | **B**renda from the **B**usiness who is keen to take the Pet Clinic in to new growth areas sees DevOps being key to that. She is extremely proud of her two golden retrievers Bonnie and Clyde.
+
+<img src="/images/chun.png" width="400"/> | **C**hun is a **c**oach, change facilitator and technologist. Chun is an avid outdoors person and adrenaline junky, lover of fun, hater of being on-call. She encourages teams to automate as much as possible of the build, test, package and deploy process to ensure that the systems were created with a LessOps model in mind.
```
Comment (pberthonneau) [inline blog/_posts/2020-04-14-introducing-online-devops-dojo.md:56]:
```suggestion
![Chun](/images/chun.png) | **C**hun is a **c**oach, change facilitator and technologist. Chun is an avid outdoors person and adrenaline junky, lover of fun, hater of being on-call. She encourages teams to automate as much as possible of the build, test, package and deploy process to ensure that the systems were created with a LessOps model in mind.

```

File: blog/_posts/2020-04-14-introducing-online-devops-dojo.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+<img src="/images/dan.png" width="400"/> | **D**an, has worked as a senior developer for a number of years, most recently for First Order Incorporated a role he left due to the excessive travel requirements. Dan's workmates still fondly recall the fun and games which ensued when Shelob, his pet tarantula, got loose on *"Bring Your Pets to Work"* day.
```
Comment (pberthonneau) [inline blog/_posts/2020-04-14-introducing-online-devops-dojo.md:57]:
```suggestion
![Dan](/images/dan.png) | **D**an, has worked as a senior developer for a number of years, most recently for First Order Incorporated a role he left due to the excessive travel requirements. Dan's workmates still fondly recall the fun and games which ensued when Shelob, his pet tarantula, got loose on *"Bring Your Pets to Work"* day.

```

File: blog/_posts/2020-04-14-introducing-online-devops-dojo.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+<img src="/images/adam.png" width="400"/> | **A**dam is a System **A**dministrator transitioning to the role of Site Reliability Engineer (SRE). Adam is working closely with Selma to ensure the security of the company's software and applications. Adam suspects that Hal may have an inside man. Adam has a particular set of skills so he is working on a plan to look for the hacker, to find the hacker, to ... the hacker.
```
Comment (pberthonneau) [inline blog/_posts/2020-04-14-introducing-online-devops-dojo.md:58]:
```suggestion
![Adam](/images/adam.png) | **A**dam is a System **A**dministrator transitioning to the role of Site Reliability Engineer (SRE). Adam is working closely with Selma to ensure the security of the company's software and applications. Adam suspects that Hal may have an inside man. Adam has a particular set of skills so he is working on a plan to look for the hacker, to find the hacker, to ... the hacker.

```

**Pull request merged**